web/src/components/molecules/Asset/Asset/AssetHeader/index.tsx

```diff
@@ -0,0 +1,47 @@
+import styled from "@emotion/styled";
+import Button from "@reearth-cms/components/atoms/Button";
+
+type AssetHeaerProps = {
```

Comment:
AssetHeaerProp -> AssetHeaderProps
But actually, since they are the only props in this file, you can just name them "Props" for simplicity if you want!